src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/PropertyTabAttribute.cs

```diff
@@ -150,6 +153,7 @@ public override bool Equals(object other)
             return false;
         }
 
+        // TODO-NULLABLE: other should most likely be nullable but will throw NRE
```

Comment:
As with my other comment, at this point let's just open issues for such things rather than adding more TODO-NULLABLE comments.